openprescribing/frontend/tests/functional/test_charts.py

```diff
@@ -65,3 +65,40 @@ def test_nothing_hidden_by_default(self):
         xlabels = self.find_by_xpath(
             "//*[contains(@class, 'highcharts-xaxis-labels')]")
         self.assertNotIn('GREEN', xlabels.text)
+
+
+class AnalyseSummaryTotalsTest(SeleniumTestCase):
+
+    def test_summary_totals_on_analyse_page(self):
+        self.browser.get(
+            self.live_server_url +
+            '/analyse/#org=CCG&numIds=0212000AA')
+        expected = {
+            'panel-heading': (
+                u'Total prescribing for Rosuvastatin Calcium across all '
+                u'CCGs in NHS England'
+            ),
+            'js-selected-month': u"Sep '16",
+            'js-financial-year-range': u"Apr—Sep '16",
+            'js-year-range': u"Oct '15—Sep '16",
+            'js-cost-month-total': u'29,720',
+            'js-cost-financial-year-total': u'178,726',
+            'js-cost-year-total': u'379,182',
+            'js-items-month-total': u'1,669',
+            'js-items-financial-year-total': u'9,836',
+            'js-items-year-total': u'20,622',
+        }
+        for classname, value in expected.items():
+            selector = (
+                '//*[@id="{id}"]'
+                '//*[contains(concat(" ", @class, " "), " {classname} ")]'.format(
```

Comment:
Do we have to split long strings like this?  I find it makes the code harder to read.

Reply:
(I think I prefer it to a long line... though would be interesting to have the debate!)

Reply:
I'm with Seb (unsurprisingly, given that I wrote it!). Particularly given that it's broken on a logical boundary. I don't know if xpath has an equivalent to regex verbose mode where whitespace in the expression is ignored but if so I'd be tempted to use that and make the whole thing a triple-quoted string.

Reply:
OK!  I think my main problem is that I'm not used to it.